examples/multipass/app/components/Cart.tsx

```diff
@@ -9,6 +9,7 @@ import {
 import type {CartLineUpdateInput} from '@shopify/hydrogen/storefront-api-types';
 import {Link} from '@remix-run/react';
 import type {CartApiQueryFragment} from 'storefrontapi.generated';
+import type {PartialDeep} from 'type-fest';
```

Comment:
We don't have type-fest as a dependency of multipass nor skeleton template... should we add it? Or just change this specific type?